ui/src/diary/QuipReactions/QuipReactions.tsx

```diff
@@ -0,0 +1,67 @@
+import EmojiPicker from '@/components/EmojiPicker';
+import AddReactIcon from '@/components/icons/AddReactIcon';
+import { useDiaryState } from '@/state/diary';
+import { NoteCork } from '@/types/diary';
+import _ from 'lodash';
+import React, { useCallback, useState } from 'react';
+import { useParams } from 'react-router';
+import QuipReaction from './QuipReaction';
+
+interface QuipReactionsProps {
+  whom: string;
+  cork: NoteCork;
+  time: string;
+  noteId: string;
+}
+
+export default function QuipReactions({
+  whom,
+  cork,
+  time,
+  noteId,
+}: QuipReactionsProps) {
+  const [pickerOpen, setPickerOpen] = useState(false);
```

Comment:
just a meta comment, we probably should move this up and out (and the chat one) later, to a more global state at some point, because any time these remount the picker goes away which unfortunately happens a lot